packages/Chem/src/package.ts

```diff
@@ -1354,4 +1354,24 @@ export async function demoScaffold(): Promise<void> {
   _demoScaffoldTree();
 }
 
+//name: validateMolecule
+//input: string s
+//output: object result
+export function validateMolecule(s: string): DG.chem.IValidationRes {
+  let logHandle;
+  let mol;
```

Comment:
Do not declare variables as "let" when you know the type

Reply:
done